Src/WitsmlExplorer.Api/Jobs/DeleteRiskJob.cs

```diff
@@ -1,9 +1,11 @@
+using System.Collections.Generic;
 using WitsmlExplorer.Api.Jobs.Common;
+using WitsmlExplorer.Api.Models;
 
```

Comment:
`System.Collections.Generic` and `WitsmlExplorer.Api.Models`seems to be redundant and can be removed if that is the case here